src/model/loader.nit

```diff
@@ -58,6 +58,15 @@ redef class AppConfig
 		var html = proc.process(content).write_to_string
 
 		var track = new Track(title_id, title, html)
+
+		var ls = ini["languages"]
+		if ls != null then
```

Comment:
What should we do if the `languages` setting of a track is unavailable? Shoot an error or consider all languages are accepted ?